skimage/exposure/tests/test_exposure.py

```diff
@@ -255,10 +266,13 @@ def test_rescale_shrink():
     assert_array_almost_equal(out, [0, 0.5, 1])
 
 
-def test_rescale_in_range():
-    image = np.array([51., 102., 153.])
+@pytest.mark.parametrize("dtype", [np.float16, np.float32, np.float64])
```

Comment:
There's a mix of `@pytest.mark.parametrize('dtype', [np.float16, np.float32, np.float64])` and `@pytest.mark.parametrize("dtype", [np.float16, np.float32, np.float64])` in these new/updated tests... :frowning:

Reply:
I will update to consistently use single quotes for the new lines added in this PR. There is a mixture of single and double quotes elsewhere in the file, but probably not worth changing the unrelated stuff in this PR.